source/ocsp-support/ocsp-support.rst

```diff
@@ -339,8 +364,10 @@ a response and the OCSP responder is down.
 Drivers that fail “Soft Fail Test” MUST document that their driver’s
 TLS library utilizes “hard fail” behavior in the case of an
 unavailable OCSP responder in contrast to the mongo shell and drivers
-that utilize “soft-fail” behavior. Such drivers MUST also document the
-potential backwards compatibility issues as noted in the `Backwards
+that utilize “soft fail” behavior. They also MUST document the change
+in defaults for the applicable options (see `MongoClient
+Configuration`_). Such drivers MUST also document the potential
+backwards compatibility issues as noted in the `Backwards
```

Comment:
AFAIK, no driver has had a stable release with OCSP. Why would any driver need to document a BC issue?

Reply:
So the line about drivers documenting potential backwards breaking changes was originally in the spec because we were okay with drivers with low amounts of control over their OCSP implementation defaulting to turning OCSP on even if they were hard-failing. Such a change would also be a backwards breaking change for applications deployed behind a restrictive allow-list.

Now that we're explicitly disallowing drivers from defaulting to OCSP to on when they exhibit hard-fail, rewording this section a bit for clarity is probably a reasonable idea. I've taken Divjot's suggestion and tweaked it mildly :) 

Some extra .NET specific history that inspired the original wording:
Historically, the .NET driver changed the default for revocation checking several times, and when the default changes from false to true, it can cause breaking changes in the case of applications deployed behind a restrictive allow-list because the driver defaults to hard-fail (and we've had a HELP ticket come in about such a case).